docs/source/conf.py

```diff
@@ -52,7 +52,10 @@ def _parse_release_as_version(rls):
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.todo'
+    #'sphinx.ext.todo'
+    #'sphinx.ext.autodoc',  # For pulling in docstrings
+    #'sphinx.ext.doctest',  # For testing code examples in documentation
+    #'sphinx.ext.napoleon',  # For Google and NumPy-style docstrings
```

Comment:
Was there a reason why `sphinx.ext.todo` was removed? Also, I don't think it's necessary to leave in commented out code.